charts/osm-arc/values.yaml

```diff
@@ -18,11 +18,11 @@ osm:
     image:
       registry: mcr.microsoft.com/oss/openservicemesh
     enforceSingleMesh: true
-    enablePrometheus: false
+    deployPrometheus: false
```

Comment:
why do we need to set this as false? It's already false upstream: https://github.com/openservicemesh/osm/blob/main/charts/osm/values.yaml#L117

Reply:
We had this in past releases of osm-arc as well. Some fields (like deployPrometheus, deployJaeger, and enforceSingleMesh) we know we want to be false/true for every osm-arc release, so that even if future upstream releases change any of these values, we still want to enforce that these values remain false/true for osm-arc. Otherwise we will need to delete/re-add these fields in our values.yaml if the default values upstream change.